src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -208,6 +215,35 @@
  * }
  * </pre>
  * <p>
+ * To configure the check to allow methods which contain a specified comment text
+ * pattern in their javadoc to be designed for extension.
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;DesignForExtension&quot;&gt;
+ *   &lt;property name=&quot;requiredJavadocPhrase&quot;
+ *     value=&quot;This implementation&quot;/&gt;
```

Comment:
there will be questions from users on how we treat match on multi lines.
Please add fee words on this in property description and please add example 
```
 *   /&#42;&#42; This
 *   &#42; implementation ...
```

Reply:
Done.